perses/distributed/relative_setup.py

```diff
@@ -312,7 +312,7 @@ class NonequilibriumSwitchingFEP(object):
 
     def __init__(self, topology_proposal, pos_old, new_positions, use_dispersion_correction=False,
                  forward_functions=None, ncmc_nsteps=100, nsteps_per_iteration=1, concurrency=4, platform_name="OpenCL",
-                 temperature=300.0 * unit.kelvin, trajectory_directory=None, trajectory_prefix=None):
+                 temperature=300.0 * unit.kelvin, trajectory_directory=None, trajectory_prefix=None, atom_selection="not water"):
```

Comment:
Is the atom_selection an md_traj selection string of some kind? Might be helpful to add a comment about it, or perhaps add to docstring.

Reply:
Ok, adding docstring. Just noticed it was missing. There's an issue open to document this class better, but will add docstring here and address other docs in another PR.